### ISSUE TITLE
Background makes text hard to read in Firefox

### ISSUE THREAD

**Issue by yuriks** [metakirby5/yorha#2]
Some background effect is not applied properly on Firefox (FF 52.0.2, Windows 10), making the text very hard to read.

Screenshot:
![image](https://cloud.githubusercontent.com/assets/341401/25061750/fee44a10-2171-11e7-90c3-7056543dc16e.png)

Looks normal on Chrome on the same machine.

**Comment by metakirby5** [comment]
Odd, I can't seem to reproduce it on the same Firefox version. Perhaps it's a Windows-specific problem, in which case I probably cannot fix this myself.

![](https://www.dropbox.com/s/s54os3buo6tjv3y/Screenshot%202017-04-15%2001.50.26.png?raw=1)

**Comment by yuriks** [comment]
I was able to work around it by changing the `background-image` to:
```
linear-gradient(to right, rgba(204, 200, 177, 1.0) 1px, rgba(204, 200, 177, 0.0) 1px),
linear-gradient(to bottom, rgba(204, 200, 177, 1.0) 1px, rgba(204, 200, 177, 0.0) 1px)
```
This leads me to believe that the issue is that Firefox on Windows (maybe only under certain GPUs?) isn't using pre-multiplied alpha during compositing. So if the gradients aren't pixel perfect, you get a darkened blending artifact.

Changing `background-size` to be px-based instead of rem-based also makes the issue go away as long as the zoom level is 100%.

**Comment by metakirby5** [comment]
Ah, that makes sense. Would you mind opening a PR for that?

**Comment by yuriks** [comment]
I could try making a PR, but I don't know Styl and have no way of building the CSS to see if it works. I was just playing around in the Firefox Developer tools...

**Comment by metakirby5** [comment]
Ok no problem -- leave it to me :) I just thought you had everything set up already, and PRs are the best way to give a contributor the credit they deserve.

**Comment by metakirby5** [comment]
Mind testing out the `fix-ff-bg` branch for #3? Opening the `index.html` should suffice.

**Comment by yuriks** [comment]
Working fine!

![image](https://cloud.githubusercontent.com/assets/341401/25128131/6580b7d6-23ed-11e7-82d3-b2bf77272f70.png)
